src/index.js

```diff
@@ -1,7 +1,369 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
+import React from "react";
+import ReactDOM from "react-dom";
+import Map from "./Map.js";
+import countyMap from "./assets/county-map.jpg";
+import countyData from "./border";
+import leafletPip from "@mapbox/leaflet-pip";
+import L from "leaflet";
+import { ReactComponent as UpArrow } from "./assets/arrow-up.svg";
+import { ReactComponent as RightArrow } from "./assets/arrow-right.svg";
+import { ReactComponent as DownArrow } from "./assets/arrow-down.svg";
+import { ReactComponent as LeftArrow } from "./assets/arrow-left.svg";
+
+class App extends React.Component {
+ 
+  constructor(props) {
+    super(props);
+    this.countyLayer = this.makeCountyLayer();
+    this.state = {
+      startPosition: { lat: 0, lon: 0 },
+      markerPosition: { lat: 0, lon: 0 },
+      score: 100,
+      gameStarted: false,
+      startingCounty: null,
+      gameOver: false,
+      guess: "",
+      readyToGuess: false,
+      gameOverText: null,
+      gaveUp: false
+      //To do: Add an array of previous movements (as object with lat and lon) for history and breadcrumbs
+    };
+
+    this.pickRandomLatLon = this.pickRandomLatLon.bind(this);
+    this.makeCountyLayer = this.makeCountyLayer.bind(this);
+    this.checkStart = this.checkStart.bind(this);
+    this.move = this.move.bind(this);
+    this.startGame = this.startGame.bind(this);
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.showGuessForm = this.showGuessForm.bind(this);
+    this.giveUp = this.giveUp.bind(this);
```

Comment:
If you do your method definitions as arrow functions (e.g. `makeCountyLayer = () => { ... }`) then you don't have to rebind the `this` since arrow functions automatically bind `this` to the containing scope.